contracts/InterestRateModels/TwoKinksInterestRateModel.sol

```diff
@@ -0,0 +1,164 @@
+pragma solidity ^0.5.16;
```

Comment:
I'm sorry I bring this up a bit late, but do we really have to go with 0.5 here? We could create a separate interface for v8 InterestRateModel and simplify things a bit, both in terms of gas costs (thanks to immutables) and SignedSafeMath (which, AFAIU, has to be audited separately and likely affects the audit costs)

Reply:
I would prefer to use 0.8 if it's possible, for sure.

I suppose we would need a new abstract contract `InterestRateModelV8` and extend it here